### PR TITLE
Addition of product types for economic variables

### LINKED ISSUE #65: Economic variables
Dear all, 

As mentioned last Thursday, a couple of economic variables could be added. For the energy models, these variables might not be used. However, for generating output, it might be good to also place the economic output in the databases. 

**Here are a couple of my suggestions.** 

1. "Consumption" is currently only referred to as consumption in monetary units. Is this energy consumption only? 

I suggest to adjust this variable as follows: 
- Add consumption (in mln euro) also for non-energy products (e.g. agricultural products, household goods etc). 
- Differentiate between domestic, imported and total consumption in monetary units
- Differentiate between household, government consumption and gross capital formation (investments) and industry consumption. Where industry consumption is defined per industry.

Consumption|Households|Imported|<product>
Consumption|Households|Domestic|<product>
Consumption|Households|Total|<product>
Consumption|Government|Imported|<product>
Consumption|Government|Domestic|<product>
Consumption|Government|Total|<product>
Consumption|GrossCapitalFormation|Imported|<product>
Consumption|GrossCapitalFormation|Domestic|<product>
Consumption|GrossCapitalFormation|Total|<product>
Consumption|<industrie>|Imported|<product>
Consumption|<industrie>|Domestic|<product>
Consumption|<industrie>|Total|<product>

2. Output  (productivity per industry). This indicates whether a sector is benefiting (growing) under certain energy policies. This should therefore also include more industries than just the energy sector. E.g. agricultural, transport, manufacturing sectors. These can be provided in monetary units (million euros).

IndustryOutput|<industry>
ProductOutput|<industry>

3. The use of product factors: capital and labor per industry. Both in monetary units (million euro). My suggestion is: 

Capital|<industry>
Labor|<industry>

4. Price indices. The economic model allows to including prices but not priceindices. Our model does not provide prices but price-indices. A price index is relative with respect to year 2011. That is, in 2011, all price-indices are equal to 1. In later years, the price indices increase or decrease. A priceindex of 1.2 in 2020 for the agricultural sector implies that agri-food prices have increased with 20% with respect to 2011 due to the scenario. I suggest the following price indices

ProductPriceIndex|<product>
CapitalPriceIndex|<industry>
LaborPriceIndex|<industry>

The list of products and industries can be based on an aggregated product and industry definitions from EXIOBASE. Hereby a suggestion: 

<products>
pPLNT	'Crops and vegatable in agri sector'
pANIM   'Animal production'
pFORE   'Forestry and logging'
pFISH   'Fishing and aquaculture'
pFOSM   'Mining fossil fuels'
pOTHM   'Mining of metal ores and non-metallic minerals'
pFBTO   'Manufacturing of food, beverage and tobacco products'
pTXWO   'Manufacturing of textile, wood and printed products'
pCOKE   'Manufacturing of coke products'
pREFN   'Manufacturing of refined petroleum products'
pCHEM   'Manufacturing of chemicals and chemical products'
pRUBP   'Manufacturing of rubber and plastic products'
pNMMP   'Manufacturing of non-metallic mineral products'
pMETP   'Manufacturing of basic metals and metal products'
pELEC   'Manufacturing of electronic computer, optical and electrical equipment'
pMACH   'Manufacturing of machinery and equipment nec and other manufacturing'
pELEQ   'Electricity'
pTRDI   'Transmission and distribution services'
pHWAT   'Steam and hot water supply services'
pWATR   'Collected and purified water, distribution services of water'
pCONS   'Construction'
pTRAD   'Wholesale and retail trade'
pHORE   'Accomodation and food service activities'
pTRAN   'Transportation services'
pREBA   'Real estate, renting and business activities'
pRENT 	'Renting services of machinery and equipment without operator and of personal and household goods (71)' 
pPUBO   'Public administration, education, health and other activities'
pWAST   'Waste for treatment'

<Industries>
iPLNT	'Crops and vegatable in agri sector'
iANIM   'Animal production'
iFORE   'Forestry and logging'
iFISH   'Fishing and aquaculture'
iFOSM   'Mining fossil fuels'
iOTHM   'Mining of metal ores and non-metallic minerals'
iFBTO   'Manufacturing of food, beverage and tobacco products'
iTXWO   'Manufacturing of textile, wood and printed products'
iCOKE   'Manufacturing of coke products'
iREFN   'Manufacturing of refined petroleum products'
iCHEM   'Manufacturing of chemicals and chemical products'
iRUBP   'Manufacturing of rubber and plastic products'
iNMMP   'Manufacturing of non-metallic mineral products'
iMETP   'Manufacturing of basic metals and metal products'
iELEC   'Manufacturing of electronic computer, optical and electrical equipment'
iMACH   'Manufacturing of machinery and equipment nec and other manufacturing'
iELCO         'Production of electricity by coal'
iELGA          'Production of electricity by gas'
iELNU          'Production of electricity by nuclear'
iELHY          'Production of electricity by hydro'
iELWI          'Production of electricity by wind'
iELOI          'Production of electricity by petroleum and other oil derivatives'
iELBI          'Production of electricity by biomass and waste'
iELSO          'Production of electricity by solar photovoltaic or thermal'
iELOC          'Production of electricity by tide, wave, ocean'
iELGE          'Production of electricity by Geothermal'
iELNE          'Production of electricity nec'
iTRDI   'Transmission and distribution services'
iHWAT   'Steam and hot water supply services'
iWATR   'Collected and purified water, distribution services of water'
iCONS   'Construction'
iTRAD   'Wholesale and retail trade'
iHORE   'Accomodation and food service activities'
iTRAN   'Transportation services'
iREBA   'Real estate, renting and business activities'
iRENT	'Renting of machinery and equipment without operator and of personal and household goods'
iPUBO   'Public administration, education, health and other activities'
iWAST   'Waste for treatment'

### PR DESCRIPTION
**Pull request by erikfilias**
This PR is a first step towards implementing the proposal in #65, namely to add a list of products for economic variable definitions. It is based on the same procedure made in #64.

 The key point: economic variables by product types are generically given by `Consumption|Households|< product >|Imported:` without having to define all in `economy.yaml`) and the `<product>` tag will be replaced by the list given in `product_types.yaml` when importing the Python package (so that the validation continues to work as expected). 
Then, this PR will make the following changes:

- Addition of `product_types.yaml` and `industry_types.yaml` files which include a list of products and industries.
- Addition of generic variables in `economy.yaml` file.
- Addition of some code lines in the validation function to be capable to import the lists.

Also, a test is added to make sure that the correct variable and description are available via the Python package.

closes #65

### CODE REVIEW

File: nomenclature/definitions/variable/product_types.yaml [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+
+#   i.e., The following standard structure in economic variables:
+#   Consumption|Households|< product >|Imported:
+#   Consumption|Government|< product >|Domestic:
+
+<product>:
+  pPLNT:
+    description: Crops and vegatable in agri sector
+
+  pANIM:
```
Comment (erikfilias) [inline nomenclature/definitions/variable/product_types.yaml:13]: Dear @hettieboonman, I'm confused about the prefixes `p` and `n` in each product. 
Could you explain us a little about ti, please?
I'll try to put your explanation as a clarification in this `product_types.yaml` file.

**Comment by erikfilias** [comment]
Dear @hettieboonman, @sebastianzwickl , and @danielhuppmann ,

Here, I'm starting to define the generic economic variables. 
Any comment is very welcome.

**Comment by hettieboonman** [comment]
Dear Erik, 

Thanks for the work, this looks good to me! 

Greets, Hettie

**Comment by erikfilias** [comment]
> Dear Erik,
> 
> Thanks for the work, this looks good to me!
> 
> Greets, Hettie

Your welcome,
@danielhuppmann , @sebastianzwickl , could you review it, please? Thanks in advance.

File: nomenclature/__init__.py [hunk 2]
```diff
@@ -49,6 +49,10 @@ def _copy_dict(dct, description):
 
 # explode <Fuels> tags to full lists
 fuel_types = _variables.pop('<Fuel>')
+# explode <industry> tags to full lists
+industry_types = _variables.pop('<industry>')
+# explode <product> tags to full lists
+product_types = _variables.pop('<product>')
```
Comment (sebastianzwickl) [inline nomenclature/__init__.py:55]: We use here for the different tags (`<Fuel>`, `<industry>`, `<product>`) only once a capital letter.

File: nomenclature/definitions/variable/economy/economy.yaml [hunk 3]
```diff
@@ -4,6 +4,66 @@ Consumption:
...
+Consumption|Industry|<product>|Imported:
+   description: Consumption of imported <this product> by industries
+   unit: million euro/yr
+
+Consumption|Industry|<product>|Domestic:
+   description: Consumption of domestic <this product> by industries
+   unit: million euro/yr
+
+IndustryOutput|<industry>:
+   description: Total output on the <this industry> level
```
Comment (sebastianzwickl) [inline nomenclature/definitions/variable/economy/economy.yaml:40]: Maybe only `Total output on <this industry> level`

File: nomenclature/definitions/variable/economy/economy.yaml [hunk 4]
```diff
@@ -4,6 +4,66 @@ Consumption:
...
+   unit: million euro/yr
+
+ProductOutput|<product>:
+   description: Total output on the <this product> level
```
Comment (sebastianzwickl) [inline nomenclature/definitions/variable/economy/economy.yaml:44]: Maybe only `Total output on <this product> level`

File: nomenclature/definitions/variable/economy/economy.yaml [hunk 5]
```diff
@@ -4,6 +4,66 @@ Consumption:
...
+Capital|<industry>:
+   description: Total capital costs spend by <this industry>
+   unit: million euro/yr
+
+Labor|<industry>:
+   description: Total wages spend by <this industry>
+   unit: million euro/yr
+
+ProductPriceIndex|<product>:
+   description: Price index on the <this product> level, relative to base year 2011
```
Comment (sebastianzwickl) [inline nomenclature/definitions/variable/economy/economy.yaml:56]: Maybe only `Price index on <this product> level, relative to base year 2011`

**Review by sebastianzwickl — COMMENTED** [review]
Just a few side remarks from my side.

File: nomenclature/__init__.py (hunk 2 above)
Comment (erikfilias) [reply nomenclature/__init__.py]: Got it! Thx.

File: nomenclature/definitions/variable/economy/economy.yaml (hunk 3 above)
Comment (erikfilias) [reply nomenclature/definitions/variable/economy/economy.yaml]: Got it!

File: nomenclature/definitions/variable/economy/economy.yaml (hunk 4 above)
Comment (erikfilias) [reply nomenclature/definitions/variable/economy/economy.yaml]: Got it!

File: nomenclature/definitions/variable/economy/economy.yaml (hunk 5 above)
Comment (erikfilias) [reply nomenclature/definitions/variable/economy/economy.yaml]: Got it!

**Comment by erikfilias** [comment]
> Just a few side remarks from my side.

Thanks!

**Comment by erikfilias** [comment]
Thank you! Please, check it.

**Review by sebastianzwickl — APPROVED** [review]

File: nomenclature/definitions/variable/economy/economy.yaml [hunk 6]
```diff
@@ -4,6 +4,66 @@ Consumption:
    description: Total consumption of all goods, by all consumers in a region
    unit: billion US$2010/yr
 
+Consumption|Households|<Product>|Imported:
```
Comment (danielhuppmann) [inline nomenclature/definitions/variable/economy/economy.yaml:7]: Should there also be aggregate categories `Consumption|Households`, `Consumption|Government` and `Consumption|Industry`?

File: nomenclature/definitions/variable/economy/economy.yaml [hunk 7]
```diff
@@ -4,6 +4,66 @@ Consumption:
...
+
+Consumption|Government|<Product>|Imported:
+   description: Consumption of imported <this product> by government
+   unit: million euro/yr
+
+Consumption|Government|<Product>|Domestic:
+   description: Consumption of domestic <this product> by government
+   unit: million euro/yr
+
+Consumption|GrossCapitalFormation|<Product>|Imported:
```
Comment (danielhuppmann) [inline nomenclature/definitions/variable/economy/economy.yaml:23]: Please refactor to `Gross Capital Formation` in line with the [variable naming convention](https://github.com/openENTRANCE/nomenclature/tree/master/nomenclature/definitions/variable#naming-convention-for-variables)

File: nomenclature/__init__.py [hunk 8]
```diff
@@ -69,6 +73,29 @@ def _copy_dict(dct, description):
                     _key_ccs = f'{_key}|{sub}'
                     _description_ccs = f'{_description} {desc}'
                     variables[_key_ccs] = _copy_dict(value, _description_ccs)
+
+    # if the key contains the tag, loop over all industry types to add mapping
```
Comment (danielhuppmann) [inline nomenclature/__init__.py:77]: this piece of code is repetitive and could be refactored like:
```
key_types = [
    ('<Fuel>', fuel_types),
    ('<Industry>', industry_types),
    ('<Product>', product_types)
]
for k, types in key_types:
    if k in key:
        for f, attr in types():
            ...
```

File: nomenclature/definitions/variable/economy/economy.yaml [hunk 9]
```diff
@@ -4,6 +4,66 @@ Consumption:
...
+
+Consumption|Industry|<Product>|Imported:
+   description: Consumption of imported <this product> by industries
+   unit: million euro/yr
+
+Consumption|Industry|<Product>|Domestic:
+   description: Consumption of domestic <this product> by industries
+   unit: million euro/yr
+
+IndustryOutput|<Industry>:
```
Comment (danielhuppmann) [inline nomenclature/definitions/variable/economy/economy.yaml:39]: Please rename to `Output|Industry|<Industry>`, and is there a need for aggregate industrial output as well?

File: nomenclature/definitions/variable/economy/economy.yaml [hunk 10]
```diff
@@ -4,6 +4,66 @@ Consumption:
...
+   description: Total output on <this industry> level
+   unit: million euro/yr
+
+ProductOutput|<Product>:
```
Comment (danielhuppmann) [inline nomenclature/definitions/variable/economy/economy.yaml:43]: Please rename to `Output|Product|<Product>`, and is there a need for aggregate product output as well?

File: nomenclature/definitions/variable/economy/economy.yaml [hunk 11]
```diff
@@ -4,6 +4,66 @@ Consumption:
...
+   description: Total output on <this product> level
```
Comment (danielhuppmann) [inline nomenclature/definitions/variable/economy/economy.yaml:44]: Is the output really happening *on* a *level*?

File: nomenclature/definitions/variable/industry_types.yaml [hunk 12]
```diff
@@ -0,0 +1,122 @@
+# This list of industries will replace any occurrence of `<industry>`
+# in a variable-definition yaml file
+
+
+#   i.e., The following standard generical structure in economic variables:
+#   CapitalPriceIndex|<industry>:
+#   LaborPriceIndex|<industry>:
+
+<Industry>:
+  iPLNT:
```
Comment (danielhuppmann) [inline nomenclature/definitions/variable/industry_types.yaml:10]: This is not intuitive to non-experts, any chance to use slightly more descriptive names?

File: nomenclature/definitions/variable/industry_types.yaml [hunk 13]
```diff
@@ -0,0 +1,122 @@
...
+    description: Production of electricity by solar photovoltaic or thermal
+
+  iELOC:
+    description: Production of electricity by tide, wave, ocean
+
+  iELGE:
+    description: Production of electricity by Geothermal
+
+  iELNE:
+    description: Production of electricity nec
```
Comment (danielhuppmann) [inline nomenclature/definitions/variable/industry_types.yaml:89]: What is "nec"?

File: nomenclature/definitions/variable/economy/economy.yaml [hunk 14]
```diff
@@ -4,6 +4,66 @@ Consumption:
...
+
+Capital|<Industry>:
+   description: Total capital costs spend by <this industry>
+   unit: million euro/yr
+
+Labor|<Industry>:
+   description: Total wages spend by <this industry>
+   unit: million euro/yr
+
+ProductPriceIndex|<Product>:
```
Comment (danielhuppmann) [inline nomenclature/definitions/variable/economy/economy.yaml:55]:
```suggestion
Product Price Index|<Product>:
```

File: nomenclature/definitions/variable/economy/economy.yaml [hunk 15]
```diff
@@ -4,6 +4,66 @@ Consumption:
...
+   description: Price index on <this product> level, relative to base year 2011
+   unit: index equals 1 in 2011, the reference year
+
+CapitalPriceIndex|<Industry>:
```
Comment (danielhuppmann) [inline nomenclature/definitions/variable/economy/economy.yaml:59]:
```suggestion
Capital Price Index|<Industry>:
```

File: nomenclature/definitions/variable/economy/economy.yaml [hunk 16]
```diff
@@ -4,6 +4,66 @@ Consumption:
...
+   description: Price index of capital from <this industry>, relative to base year 2011
+   unit: index equals 1 in 2011, the reference year
+
+LaborPriceIndex|<Industry>:
```
Comment (danielhuppmann) [inline nomenclature/definitions/variable/economy/economy.yaml:63]:
```suggestion
Labor Price Index|<Industry>:
```

**Review by danielhuppmann — CHANGES_REQUESTED** [review]
thanks @erikfilias for adding the variables requested by @hettieboonman - a number of requests for clarifications and suggestions for improved implementation inline.

File: nomenclature/definitions/variable/economy/economy.yaml (hunk 6 above)
Comment (erikfilias) [reply nomenclature/definitions/variable/economy/economy.yaml:7]: To maintain the convention. I decided to add it.

File: nomenclature/definitions/variable/economy/economy.yaml (hunk 7 above)
Comment (erikfilias) [reply nomenclature/definitions/variable/economy/economy.yaml]: Got it

File: nomenclature/__init__.py (hunk 8 above)
Comment (erikfilias) [reply nomenclature/__init__.py]: Thanks, I got it.

File: nomenclature/definitions/variable/economy/economy.yaml (hunk 9 above)
Comment (erikfilias) [reply nomenclature/definitions/variable/economy/economy.yaml]: @hettieboonman, any comment?

File: nomenclature/definitions/variable/economy/economy.yaml (hunk 10 above)
Comment (erikfilias) [reply nomenclature/definitions/variable/economy/economy.yaml]: @hettieboonman, any comment?

File: nomenclature/definitions/variable/economy/economy.yaml (hunk 11 above)
Comment (erikfilias) [reply nomenclature/definitions/variable/economy/economy.yaml:56]: @hettieboonman, could you bring us an additional comment to extend the description of the `ProductOutput|<Product>` variable, please?

File: nomenclature/definitions/variable/industry_types.yaml (hunk 12 above)
Comment (erikfilias) [reply nomenclature/definitions/variable/industry_types.yaml:10]: @hettieboonman, could you bring us an additional comment to rename the `iPLNT` industry -type, please?

File: nomenclature/definitions/variable/industry_types.yaml (hunk 13 above)
Comment (erikfilias) [reply nomenclature/definitions/variable/industry_types.yaml:89]: I've just changed the list according to request from @hettieboonman in #65 , and this variable is not considered.

File: nomenclature/definitions/variable/economy/economy.yaml (hunk 6 above)
Comment (danielhuppmann) [reply nomenclature/definitions/variable/economy/economy.yaml:19]: Thanks!

File: nomenclature/definitions/variable/industry_types.yaml [hunk 17]
```diff
@@ -0,0 +1,65 @@
...
+
+<Industry>:
+  iAGRI:
+    description: 'Agriculture'
+
+  iCOAL:
+    description: 'Coal extraction'
+
+  iCOIL:
+    description: 'crude oil and natural gas extraction'
```
Comment (danielhuppmann) [inline nomenclature/definitions/variable/industry_types.yaml:17]: minor consistency update
```suggestion
    description: 'Crude oil and natural gas extraction'
```

File: nomenclature/tests/test_core.py [hunk 18]
```diff
@@ -19,6 +19,21 @@ def test_variables_fuel_types():
...
+    obs = nc.variables['Capital|iAGRI']
+    exp = 'Total capital costs spend by agriculture'
+    assert obs['description'] == exp
+
+
+def test_variables_product_types():
+    # check that exploding of <product> to procuts works
+    obs = nc.variables['Consumption|Households|pAGRI|Imported']
+    exp = 'Consumption of imported agriculture by ' \
+          'households'
```
Comment (danielhuppmann) [inline nomenclature/tests/test_core.py:33]: This should not exceed the line length...
```suggestion
    exp = 'Consumption of imported agriculture by households'
```

**Review by danielhuppmann — COMMENTED** [review]
Thanks for reworking the implementation, much more concise!

One more minor editorial change - then it's good to go from a technical point of view.

File: nomenclature/definitions/variable/industry_types.yaml (hunk 13 above)
Comment (danielhuppmann) [reply nomenclature/definitions/variable/industry_types.yaml]: Thanks for the clarification!

File: nomenclature/definitions/variable/industry_types.yaml (hunk 12 above)
Comment (danielhuppmann) [reply nomenclature/definitions/variable/industry_types.yaml]: Irrelevant per updated list of industries

**Comment by danielhuppmann** [comment]
@hettieboonman, can you please review the questions and provide more intuitive/explicit descriptions of the variables (in particular what is meant by a level)?

File: nomenclature/definitions/variable/economy/economy.yaml (hunk 11 above)
Comment (hettieboonman) [reply nomenclature/definitions/variable/economy/economy.yaml:56]: **Is the output really happening on a level?**
We could rewrite the descriptions as follows. Let me know if this is 

Output|Industry|<Industry>:
   description: Total output per <this industry>. That is, total value of production in <this industry>. 
   unit: million euro/yr

Output|Product|<Product>:
   description: Total output per <this product>. That is, total value of <this product> sold.

File: nomenclature/definitions/variable/economy/economy.yaml (hunk 10 above)
Comment (hettieboonman) [reply nomenclature/definitions/variable/economy/economy.yaml]: Sorry, I missed this question. There is indeed no need for an aggregate product output or aggregated industry output.

Comment (danielhuppmann) [reply nomenclature/definitions/variable/economy/economy.yaml]: thank you, resolving this comment to mark it as closed

File: nomenclature/definitions/variable/economy/economy.yaml (hunk 9 above)
Comment (danielhuppmann) [reply nomenclature/definitions/variable/economy/economy.yaml]: per response by @hettieboonman [here](https://github.com/openENTRANCE/nomenclature/pull/68#discussion_r514147847), resolving this comment to mark is as closed

File: nomenclature/definitions/variable/economy/economy.yaml (hunk 11 above)
Comment (erikfilias) [reply nomenclature/definitions/variable/economy/economy.yaml:56]: We already have this on [here](https://github.com/openENTRANCE/nomenclature/pull/68/files):

```
Output|Industry|<Industry>:
   description: Total output on <this industry> level
   unit: million euro/yr

Output|Product|<Product>:
   description: Total output on <this product> level
```

I'll just change the description.

**Review by danielhuppmann — CHANGES_REQUESTED** [review]
Suggestions for (I hope) more explanatory definitions of the Product based on longer email by @hettieboonman.

One more question to the macro-modellers (also @paopis) - do we want to have the monetary units indexed to a specific year, ie. real or nominal? As we do it for GDP in the IPCC-context...

**Comment by hettieboonman** [comment]
@danielhuppmann 
"One more question to the macro-modellers (also @paopis) - do we want to have the monetary units indexed to a specific year, ie. real or nominal? As we do it for GDP in the IPCC-context..."
The values as now presented by our model are nominal values or in other words 'in volume'.

**Comment by paopis** [comment]
> Suggestions for (I hope) more explanatory definitions of the Product based on longer email by @hettieboonman.
> 
> One more question to the macro-modellers (also @paopis) - do we want to have the monetary units indexed to a specific year, ie. real or nominal? As we do it for GDP in the IPCC-context...

In our model, every monetary value is divided by the year's Consumer Price Index, but it is not anchored to a base year.

File: nomenclature/definitions/variable/economy/economy.yaml [hunk 19]
```diff
@@ -4,6 +4,79 @@ Consumption:
...
+   description: Total capital costs spend by <this industry>
+   unit: million euro/yr
+
+Labor|<Industry>:
+   description: Total wages spend by <this industry>
+   unit: million euro/yr
+
+Product Price Index|<Product>:
+   description: Price index on <this product> level, relative to base year 2011
+   unit: index equals 1 in 2011, the reference year
```
Comment (danielhuppmann) [inline nomenclature/definitions/variable/economy/economy.yaml:70]: sorry, only realized this now
```suggestion
   note: The index equals 1 in 2011, the reference year
   unit: ''
```

File: nomenclature/definitions/variable/economy/economy.yaml [hunk 20]
```diff
@@ -4,6 +4,79 @@ Consumption:
...
+
+Capital Price Index|<Industry>:
+   description: Price index of capital from <this industry>, relative to base year 2011
+   unit: index equals 1 in 2011, the reference year
```
Comment (danielhuppmann) [inline nomenclature/definitions/variable/economy/economy.yaml:74]:
```suggestion
   note: The index equals 1 in 2011, the reference year
   unit: ''
```

File: nomenclature/definitions/variable/economy/economy.yaml [hunk 21]
```diff
@@ -4,6 +4,79 @@ Consumption:
...
+
+Labor Price Index|<Industry>:
+   description: Price index of wages from <this industry>, relative to base year 2011
+   unit: index equals 1 in 2011, the reference year
```
Comment (danielhuppmann) [inline nomenclature/definitions/variable/economy/economy.yaml:78]:
```suggestion
   note: The index equals 1 in 2011, the reference year
   unit: ''
```

**Review by danielhuppmann — CHANGES_REQUESTED** [review]
Apologies, just now realized that the unit wasn't going to work with the descriptive text. Please merge my suggestions, then it's good to go.

**Review by danielhuppmann — APPROVED** [review]
Thanks @erikfilias and @hettieboonman for this extension of the variable list

**Pull request merged**